### DISCUSSION TITLE
New Major Release when ?

### DISCUSSION THREAD

**Discussion by ErwanMAS** [Mailu/Mailu#3700] [General]
I saw many small changes , that generate many minor release 2024.06.x .

When the next main release will happen  ?

**Comment by nextgens** [comment]
Minor releases are backported fixes; Major releases bring in new features or otherwise major changes.

We have little to nothing on our master branch that would warrant one for now.

**Comment by MarcosLealDev** [comment]
How would we go to apply the minor releases when we use docker?  Should we recreate the images by hand? It's been a long time without an update with docker.

**Comment by ErwanMAS** [comment]
@MarcosLealDev instructions from https://mailu.io/2024.06/maintain.html#upgrading-the-mail-server are enough to do the minor upgrade . 
`docker compose pull ` full fetch the last version of each image . 

if you run `docker image ls`  you can see that docker pull the last version ( see column created )
